developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -132,6 +132,27 @@ All rake tasks are available with the following command:
 bundle exec rake -T
 ....
 
+=== Running tests in Foreman
+Make sure to run tests from the Foreman directory. You can pass an argument to specify the path to a directory or to a file.
+
+In order to run rubocop test, use the following command:
+[source, ruby]
+....
+bundle exec rubocop [<path>]
+....
+
+To run Foreman's tests:
+[source, ruby]
```

Comment:
```suggestion
[source, shell]
```

Reply:
@ekohl thanks again for your comments. I updated the commit with the fixes to your comments. 
Let me know if any more changes are required.